### ISSUE TITLE
"Error: " When updating mods

### ISSUE THREAD

**Issue by fcpwiz** [TechnicPack/TechnicSolder#390]
Hey,

When I'm going down through a build and updating many mods, sometimes I get a red box saying "Error: " Here is a screenshot: ![screen shot 2015-05-06 at 5 01 11 pm](https://cloud.githubusercontent.com/assets/2921172/7503662/71010302-f412-11e4-9efb-2faa7d96d61a.png)

This has been happening since Solder v0.7.

**Comment by king601** [comment]
Some of my staff for my packs are seeing this issue on Safari on Mac OS X 10.9.5

Two of my other users are fine on Windows with Firefox and Chrome

Just thought maybe that would help debug this more? We've tried clearing history / cache and it does work on Google Chrome on the Mac...

**Comment by fcpwiz** [comment]
@king601 Yeah, I'm using OS X 10.10.4 and Safari 6 and the error still occurs. I really hope this gets fixed. It's incredibly annoying.

**Comment by king601** [comment]
I also confirmed it does show Error on the same page you showed. We were having problems on the add mod version page where we add the mod to the solder repository in addition to your reported problem.

**Comment by fcpwiz** [comment]
Oh ok! I wouldn't of known about that because I have a custom SSH mod updater.

**Comment by king601** [comment]
I hate to go off topic here.. but is that open source? I recently wrote a ruby script to do my packaging for mods and right now I have to figure out an automated way to add them and was curious :)

**Comment by fcpwiz** [comment]
@king601 It's not open source, but PlowmanPlow made it, who is a very well trusted member of the Technic community. Should I ask him if I can send it to you?

**Comment by fcpwiz** [comment]
@king601 It's not open source, but PlowmanPlow made it, who is a very well trusted member of the Technic community. Should I ask him if I can send it to you?

**Comment by king601** [comment]
@fcpwiz you can ask, but I don't mind either way if they don't want to share it.
